sources/packages/backend/apps/api/src/services/notification/gc-notify-actions.service.ts

```diff
@@ -50,6 +71,8 @@ export class GCNotifyActionsService {
    */
   async sendMinistryFileUploadNotification(
     notification: MinistryStudentFileUploadNotification,
+    userId: number,
```

Comment:
Comments are missing for new method arguments.

Reply:
Added comments.